doc/problem/behavior.md

```diff
@@ -52,3 +59,40 @@ For In/Active State:
 - AS Probability: Already in basic time bin features.
 - AS Duration: Already in basic time bin features.
 - AS Rate: Use AS Number over AS Duration.
+
+
+## Data Requirements Description
+The data we have:
+- a dataframe of observations of location for each mice, (x, y, t) with \Delta t small.
+
+The data we require:
+- The above dataframe with AS/ IS properly classified at each point t for each mouse
+- The above dataframe with a classification of drinking, feeding and locomotion for each mouse at each time t
+- The above dataframe with a classification of consumption size and moving distance with each event at each time t
+- The above dataframe with a classification of strain number and mouse number at each time t (if available)
```

Comment:
We definitely already have this, it's the consumption size data we don't know if we have.